AdyenCard/Components/Stored Card/StoredCardAlertManager.swift

```diff
@@ -115,10 +115,20 @@ internal final class StoredCardAlertManager: NSObject, UITextFieldDelegate, Adye
             let details = CardDetails(paymentMethod: paymentMethod, encryptedSecurityCode: encryptedSecurityCode)
             completionHandler?(.success(details))
         } catch {
+            sendEncryptionErrorEvent()
             completionHandler?(.failure(error))
         }
     }
     
+    private func sendEncryptionErrorEvent() {
+        var errorEvent = AnalyticsEventError(
+            component: paymentMethod.type.rawValue,
+            type: .internal
+        )
+        errorEvent.code = AnalyticsConstants.ErrorCode.encryptionError.stringValue
+        context.analyticsProvider?.add(error: errorEvent)
```

Comment:
Double dot syntax is always a trigger. It is something that would make refactor in the future really hard (extracting analytics out of context for example). We can conform context to analytics handling protocol (we can create one if necessary) and just proxy all `add()` calls for now.